file-formats/susi/type/zif_aff_susi_v1.intf.abap

```diff
@@ -41,100 +42,98 @@ INTERFACE zif_aff_susi_v1
+      "! Type of authorization defaults
       "! $required
       application_type TYPE ty_type,
       "! <p class="shorttext">Maintenance Mode</p>
-      "! Maintenance mode.
+      "! Maintenance mode
       "! $values {@link zif_aff_susi_v1.data:co_maintenance_mode}
-      "! $required
+      "! $default {@link zif_aff_susi_v1.data:co_maintenance_mode.manual}
+      "! $showAlways
       maintenance_mode TYPE ty_maintenance_mode,
+      "! <p class="shorttext">Description</p>
+      "! Description of authorization default values
+      description      TYPE string,
     END OF ty_general_information.
 
   TYPES:
     "! <p class="shorttext">Authorization Value</p>
-    "! Authorization value
-    ty_val TYPE c LENGTH 40.
-
-  TYPES:
+    "! Authorization Value
+    ty_val TYPE c LENGTH 40,
     "! <p class="shorttext">Authorization Field</p>
-    "! Authorization field
+    "! Authorization Field
     ty_field TYPE c LENGTH 10.
 
   TYPES:
-    "! <p class="shorttext">Authorization Field Details</p>
-    "! Authorization field details
+    "! <p class="shorttext">Authorization Field Values </p>
+    "! Authorization field values
     BEGIN OF ty_authorization_field,
-      "! <p class="shorttext">Field</p>
-      "! Name of authorization field
-      "! $required
+      "! <p class="shorttext">Authorization Field</p>
+      "! Authorization field
       field TYPE ty_field,
       "! <p class="shorttext">From</p>
       "! From value
       "! $showAlways
-      low   TYPE ty_val,
+      low    TYPE ty_val,
       "! <p class="shorttext">To</p>
       "! To values
-      high  TYPE ty_val,
+      high   TYPE ty_val,
     END OF ty_authorization_field.
 
   TYPES:
     "! <p class="shorttext">Authorization Fields</p>
     "! Authorization fields
-    ty_authorization_fields  TYPE STANDARD TABLE OF ty_authorization_field WITH DEFAULT KEY.
+    ty_authorization_fields  TYPE STANDARD TABLE OF ty_authorization_field  WITH DEFAULT KEY.
 
   TYPES:
-    "! <p class="shorttext">Authorization Objects Details</p>
-    "! Authorization objects details
+    "! <p class="shorttext">Authorization Object Details</p>
+    "! Details of Authorization object
     BEGIN OF ty_authorization_object,
       "! <p class="shorttext">Object</p>
-      "! Name of authorization object
+      "! Name of Authorization Object
       "! $required
       object             TYPE c LENGTH 10,
       "! <p class="shorttext">Maintenance Status</p>
-      "! Maintenance status
+      "! Maintenance Status
       "! $values {@link zif_aff_susi_v1.data:co_maintenance_status}
       "! $default {@link zif_aff_susi_v1.data:co_maintenance_status.default_with_values}
-      "! $required
+      "! $showAlways
       maintenance_status TYPE ty_maintenance_status,
-      "! <p class="shorttext">Fields</p>
-      "! Authorization fields.
+      "! <p class="shorttext">Description</p>
+      "! Description of authorization defaults of object
+      description        TYPE string,
```

Comment:
Just for consistency. If we decide to change it from "description" to something else, we should change it here, too.

Reply:
I am not sure what I need to do

Reply:
This comment is related to https://github.com/SAP/abap-file-formats/pull/526#discussion_r1277264772

If we change the other location, I suggest to update both locations